sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -301,7 +303,7 @@ async function main() {
 main();
 ```
 
-### Get a certificate
+### Getting a KeyVault certificate
```

Comment:
Nit: the product is "Key Vault", not "KeyVault".

Reply:
Lots of instances all over docs that should change.

Reply:
Sounds good! I'll do these changes.